src/utils/admin.terminal.ts

```diff
@@ -0,0 +1,35 @@
+import yargs from 'yargs';
+import mongoose from 'mongoose';
+import 'dotenv/config';
+import { exit } from 'process';
+import { AdminSchema } from '../schemas/admin.schema';
+import { hashPassword } from './hashPass.util';
+
+const { email, username, password } = yargs(process.argv.slice(3))
+  .usage('admin -e <email> -p <password>')
```

Comment:
please update this usage example to `admin -e <email> -u <username> -p <password>`